lib/bambora/v1/payment_resource.rb

```diff
@@ -14,12 +14,57 @@ def initialize(client:)
         @sub_path = '/v1/payments'
       end
 
-      def make_payment(payment_data)
+      # Make a payment with a credit card. Also aliased as +make_payment+.
+      #
+      #
+      # @example
+      #
+      #   client = Bambora::JSONClient(base_url: '...', api_key: '...', merchant_id: '...')
+      #   payments = Bambora::V1::PaymentResource(client: client)
+      #   payments.create(
+      #     {
+      #       amount: 50,
+      #       payment_method: 'card',
+      #       card: {
+      #         name: 'Hup Podling',
+      #         number: '4504481742333',
+      #         expiry_month: '12',
+      #         expiry_year: '20',
+      #         cvd: '123',
+      #       },
+      #     },
+      #   )
+      #
+      # @param payment_data [Hash] All information relevant to making a payment.
+      #
+      # @see https://dev.na.bambora.com/docs/references/payment_APIs/
+      #
+      # @see https://dev.na.bambora.com/docs/references/payment_SDKs/take_payments/?shell#
+      #
+      # @return [Hash] Indicating success or failure of the operation.
+      def create(payment_data)
         @client.request(method: :post, path: @sub_path, body: payment_data)
       end
 
-      def make_payment_with_payment_profile(customer_code:, amount:, card_id: 1, complete: false)
-        make_payment(
+      # An alias of +create+.
```

Comment:
Is the comment necessary? I feel that for anyone looking at the source code, this aliasing will already be clear.

Reply:
It's not! I thought it was, but requesting docs on this method will return the docs for `create`. I'll remove it.